garrysmod/lua/includes/util/color.lua

```diff
@@ -2,19 +2,75 @@ local COLOR = {}
 COLOR.__index = COLOR
 
 --[[---------------------------------------------------------
-	Register our metatable to make it accessible using FindMetaTable
+	Register our metatable
 -----------------------------------------------------------]]
-
 debug.getregistry().Color = COLOR
 
 --[[---------------------------------------------------------
-	To easily create a colour table
+	Local bitwise AND function to get precise values
+-----------------------------------------------------------]]
+local function bAND( x, y )

```

Comment:
Looks like that's because bit.band rounds its first argument to the nearest integer, while you were expecting it to round down (floor it).